src/db/config/babelHook.js

```diff
@@ -0,0 +1,3 @@
+require('@babel/register');
```

Comment:
We don't need this file anymore. I realised that it would mean us adding `@babel-register` to our main dependencies which is not very efficient since it is not needed for the main code execution only for setup.

Reply:
Okay.
I will make the necessary file changes and push.